packages/fluentui/e2e/tests/toolbarMenu-example.tsx

```diff
@@ -27,7 +29,13 @@ const ToolbarExampleMenuShorthand = () => {
             id: selectors.triggerButtonId,
             menu: {
               items: [
-                { key: 'play', content: 'Play', icon: 'play' },
+                {
+                  key: 'play',
+                  id: `${selectors.menuItemButtonId}-0`,
+                  content: 'Play',
+                  icon: 'play',
+                  onClick: () => buttonAfterToolbarRef.current.focus()
```

Comment:
will `onClick: buttonAfterToolbarRef.current.focus` work?

Reply:
![image](https://user-images.githubusercontent.com/22620150/76433991-5e448280-63b5-11ea-800d-8a156ca53333.png)
when I changed it doesn't look promising...

Reply:
_type_script :)